variants/STM32C0xx/C071R(8-B)T/ldscript.ld

```diff
@@ -0,0 +1,187 @@
+/*
+******************************************************************************
+**
+** @file        : LinkerScript.ld
+**
+** @author      : Auto-generated by STM32CubeIDE
+**
+** @brief       : Linker script for STM32C071RBTx Device from STM32C0 series
```

Comment:
@fpistm I tried to generate this script also for STM32C071R8Tx. I noticed that there is a bug in STCubeMx. Not sure if you can report it internally?

Basically it generated this below. Notice how it incorrectly uses 128KBytes.

```ld
** @brief       : Linker script for STM32C071R8Tx Device from STM32C0 series
**                      128KBytes FLASH
**                      24KBytes RAM
...
...
...
/* Memories definition */
MEMORY
{
  RAM    (xrw)    : ORIGIN = 0x20000000,   LENGTH = 24K
  FLASH    (rx)    : ORIGIN = 0x8000000,   LENGTH = 128K
}
```

Reply:
No need to regenerate it. The same script work for both as you set the define LD_* which is preprocess and set to the correct value during the build.

Reply:
@fpistm understood. What I meant was that I just _tried_ it to see what happens. As you can see what I pasted above in the output. STM32CubeMx gave the wrong information in `@brief` and `MEMORY` for the STM32C071R8Tx device. It gave the wrong FLASH size.